rst/topics/failover.rst

```diff
@@ -274,7 +274,19 @@ In Lua API:
 
 You could also make node unelectable in WebUI:
```

Comment:
```suggestion
You can also make a node unelectable in WebUI:
```
Please don't substitute **could** for **can** unless you're referring to the past:
1. In previous versions of the program, you could create a project in one click. - okay
2. Should you need it, you could ask for help in our Telegram chat. - not okay!

See [this page](https://www.notion.so/tarantool/Modal-verbs-in-technical-documentation-1f2b82c4f25d4e53bd3564b39c45e438) in Notion for details.